multiapps-controller-web/src/test/java/org/cloudfoundry/multiapps/controller/web/security/AuthorizationCheckerTest.java

```diff
@@ -158,6 +161,13 @@ public void testSpaceDevelopersCacheNegativeResult() {
                .getSpaceDevelopers(Mockito.eq(UUID.fromString(THIRD_SPACE_ID)));
     }
 
+    @Test
+    void testCheckPermissionsWithNonUUIDSpaceIDString() {
+        setUpMocks(true, true, null);
+
+        assertThrows(NotFoundException.class, () -> authorizationChecker.checkPermissions(userInfo, "foobar", true));
```

Comment:
Better replace `"foobar"` with something like `"non-uuid-space-id"`. Currently when I look at this method call, I am not sure what "foobar" stands for and I need to see the actual method to verify that the second param is spaceId.